groups/sig-docs/groups.yaml

```
@@ -17,14 +17,11 @@ groups:
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     owners:
-      - jorgec@vmware.com
-      - kaitlynbarnard10@gmail.com
-      - paris.pittman@gmail.com
       - killen.bob@gmail.com
-      - zcorleissen@linuxfoundation.org
+      - onlydole@gmail.com
```

```suggestion
      - tim+cncf@scalefactory.com
      - onlydole@gmail.com
```
Suggest to add @sftim as an owner for the blog mailing group

---

In our weekly blog meeting I thought he'd suggested that he be added as a member. I'll wait for him to chime in, or for someone else to second the motion before making that update.

---

Sounds good, I made the suggestion since @sftim is listed under the owners aliases for blogs
https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES#L2-L5

---

That's a good point.

---

Moved Tim to owner, and removed myself and members section. Not sure I should be there, and w/o me or Tim, the section was empty.